src/components/layout/header/About.tsx

```diff
@@ -29,6 +41,56 @@ const About = () => {
         },
     } = useAuthStore();
 
+    const {
+        info: releasesInfo,
+        fetchingInfo: fetchingReleasesInfo,
+        fetchInfo: fetchReleasesInfo,
+    } = useReleasesInfoStore();
+    const [matchingRelease, setMatchingRelease] =
+        useState<ReleaseInfoT | null>();
+
+    useEffect(() => {
+        if (isOpen && releasesInfo === null) fetchReleasesInfo();
+    }, [isOpen, fetchReleasesInfo, releasesInfo]);
```

Comment:
@Milouu I'd like to avoid unnecessary queries, but I'm worried this solution won't update when an actual backend upgrade is performed.

What I mean is it depends only on `releasesInfo` not being null but if I upgrade the backend server then my client won't know that and this won't get updated. Is that correct?